zion/src/main/scala/edu/uci/ics/cloudberry/zion/actor/ViewDataAgent.scala

```diff
@@ -27,7 +27,7 @@ class ViewDataAgent(override val dbName: String,
 }
 
 object ViewDataAgent {
-  def props(dbName: String, schema: Schema, queryParser: IQLGenerator, conn: IDataConn, config: Config)
+  def props(dbName: String, schema: AbstractSchema, queryParser: IQLGenerator, conn: IDataConn, config: Config)
```

Comment:
For all the logic about `dataSetAgent`. I need to check for the logic and give you possibility to do that.